pkg/skaffold/kubernetes/manifest/errors.go

```diff
@@ -75,7 +75,7 @@ func nsSettingErr(err error) error {
 }
 
 func nsAlreadySetErr() error {
-	err := fmt.Errorf("namespace already set in the namesapce")
+	err := fmt.Errorf("namespace already set in the namespace")
```

Comment:
should this be rephrased to something like the error code?
```
	err := fmt.Errorf("namespace field already set in the manifests")
```

Reply:
Good idea, updated the text to be as you suggest